Assets/Scripts/GameStructure/Narrative/NarrativePanelObject.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace GameStructure.Narrative
+{
+    public class NarrativePanelObject : MonoBehaviour
+    {
+        [SerializeField] private Image narrativePanelImage;
+
+        public Image NarrativePanelImage => narrativePanelImage;
+
+        public void SetActive(bool enable) => gameObject.SetActive(enable);
```

Comment:
Is this actually required?

Reply:
So actually, it's not required but I didn't to call
class.gameobject.SetActive(bool) - and then imagine we also add more logic later to the way we stop showing a panel (ending audio, cancelling text fades, etc.) - I think this is less risky. 

Happy to let @SabrinaTheCookie decide if clarity is okay here (we would obviously expand this from an expression body to a proper method if we added more logic)